src/main/java/org/jabref/logic/openoffice/oocsltext/CSLCitationOOAdapter.java

```diff
@@ -21,32 +21,32 @@
 
 public class CSLCitationOOAdapter {
 
-    private static final BibEntryTypesManager BIB_ENTRY_TYPES_MANAGER = new BibEntryTypesManager();
-    private static final CitationStyleOutputFormat FORMAT = CitationStyleOutputFormat.HTML;
+    private final CitationStyleOutputFormat format = CitationStyleOutputFormat.HTML;
+    private final BibEntryTypesManager bibEntryTypesManager = new BibEntryTypesManager();
```

Comment:
Use the already existing instance either by         BibEntryTypesManager entryTypesManager = Injector.instantiateModelOrService(BibEntryTypesManager.class);

or try to pass it down via constructor

Reply:
Done. Went with the latter.